packages/components/demo/example/example.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed, waitForAsync } from "@angular/core/testing";
+import { IonicModule } from "@ionic/angular";
+
+import { TmplExampleComponent } from "./example.component";
+
+describe("TmplExampleComponent", () => {
```

Comment:
nit(non-blocking): it would be good to check if this code actually runs, and possibly add a comment to explain how to test this component in isolation (filter tests to just include this). If possible it would also be good to update the test to pass a mock value for `_row` so that it can test against how the component would normally be mounted.